src/plugins_exts/schema_mount.c

```diff
@@ -692,7 +692,7 @@ lyplg_ext_schema_mount_get_parent_ref(const struct lysc_ext_instance *ext, struc
 {
     LY_ERR res;
```

Comment:
We use `ret` or `rc`, we definitely do not need another variable `res` for this purpose, enough that we have to. I write `rc` in all new code.